bin/nwbuild.cjs

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env node
 
-const yargs = require("yargs/yargs");
-const { hideBin } = require("yargs/helpers");
+/**
+ * @file    The binary definition file used by CLI users, such as "npx nwbuild"
+ * @author  ayushmxn
```

Comment:
I think the point of `@author` is to know who wrote the file originally when on a team, so you know who to go talk to about it if you have questions or something? I don't know. I don't really think it offers a ton of value in open source. The more important thing is the `@file` description that summarizes what the file is for/does. That's a huge time save for someone new to the code base just trying to find where to add or change something. I'll see if I can change the linting rule to require the `@file` but not the `@author`.

Reply:
Bless, thanks!